vscode/microsoft-kiota/src/steps.ts

```diff
@@ -301,12 +302,13 @@ export async function generateSteps(existingConfiguration: Partial<GenerateState
                 shouldResume: shouldResume
             });
         }
+        state.pluginTypes = ['ApiPlugin'];
```

Comment:
Confirm that the steps total for the ApiPlugin type would be 3 and not 4. Since we have reduced number of steps/quickpick items to be shown for it. Same for the [OpenAI] and generationtype [apimanifest].